fastlane/Fastfile

```diff
@@ -452,13 +452,6 @@ platform :ios do
     end
   end
 
-  lane :replace_assets do
-    if ENV['REPLACE_ASSETS'] == 'true'
-      sh 'cp -R ../dist/assets/release/icons/ios/* ../ios/Mattermost/Images.xcassets/AppIcon.appiconset/'
-      sh 'cp -R ../dist/assets/release/splash_screen/ios/* ../ios/SplashScreenResource/'
```

Comment:
@migbot I've never used fastlane, so my knowledge of it is limited. I removed the  `dist/assets/release/splash_screen/*` resources because they were duplicated in the `ios/` and `android/` directories and I wasn't sure how they were being used. Is `:replace_assets` only used sometimes? There might be different images in `dist/` versus `ios/`? Does it matter that the launch icon is now in `Images.xcassets` instead of `SplashScreenResource/`? I don't mind making the new versions of the images once I get the size that they should be rendered. If someone else can make them though that would be great! It's just the icon in dark and light for ios and android. The background is set elsewhere. Apologies for my lack of knowledge!

Reply:
@migbot Okie doke, I restored the fastlane assets and scaled the logo down to 108 :)